src/Features/LanguageServer/Protocol/Features/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.cs

```diff
@@ -232,17 +190,6 @@ private static DiagnosticAnalysisResult GetResultOrEmpty(ImmutableDictionary<Dia
             return DiagnosticAnalysisResult.CreateEmpty(projectId, version);
         }
 
-        public void LogAnalyzerCountSummary()
-            => _telemetry.ReportAndClear(_correlationId);
```

Comment:
not sure if this was somethign we ever used.  if so, we'll need some other mechanism to report this.

Reply:
We would need LSP pull diagnostic APIs to thread in the request kind, such as background analysis, squiggles, tagger, etc. to be able to log any analyzer telemetry for background analysis. Definitely seems doable as a follow-up work item if we wish to continue logging this telemetry.